docs/index.rst

```diff
@@ -33,6 +33,7 @@ HDL Reference Designs
    ADAQ7980-SDZ <projects/adaq7980_sdz/index>
    CN0540 <projects/cn0540/index>
    CN0561 <projects/cn0561/index>
+   PULSAR_ADC <projects/pulsar_adc_pmdz/index>
```

Comment:
Can you please change PULSAR_ADC into PULSAR-ADC-PMDZ, for consistency reasons? See https://github.com/analogdevicesinc/hdl/blob/main/docs/index.rst?plain=1#L30

Reply:
Done.